pkgs/applications/blockchains/digibyte.nix

```diff
@@ -1,33 +1,21 @@
-{ lib, stdenv
-, fetchFromGitHub
-, openssl
-, boost
-, libevent
-, autoreconfHook
-, db4
-, pkg-config
-, protobuf
-, hexdump
-, zeromq
-, withGui
-, qtbase ? null
-, qttools ? null
-, wrapQtAppsHook ? null
+{ lib, stdenv, fetchFromGitHub, openssl, boost, libevent, autoreconfHook
+, db4, pkg-config, protobuf, hexdump, zeromq, withGui ? true, qtbase ? null
+, qttools ? null, wrapQtAppsHook ? null
```

Comment:
```suggestion
, db4, pkg-config, protobuf, hexdump, zeromq, withGui ? true, qtbase
, qttools, wrapQtAppsHook
```

Reply:
The following lines exist in [all-packages.nix](https://raw.githubusercontent.com/NixOS/nixpkgs/34425dac6f28df548ef378096a09e3389791de5a/pkgs/top-level/all-packages.nix):
```nix
digibyte = libsForQt514.callPackage ../applications/blockchains/digibyte.nix { withGui = true; };
digibyted = callPackage ../applications/blockchains/digibyte.nix { withGui = false; };
```

As a result, `qtbase` [can be null](https://github.com/NixOS/nixpkgs/runs/3135375532?check_suite_focus=true#step:4:113). What do you think is the best way to proceed?

Reply:
It cannot be null. The package is still defined.